pom.xml

```diff
@@ -8,7 +8,7 @@
 
     <groupId>org.openhab.deps</groupId>
     <artifactId>pom</artifactId>
-    <version>1.0.30</version>
+    <version>1.0.31</version>
```

Comment:
You have to change the project version throughout the repo, see our past updates.